src/Control/Carrier/Telemetry/Types.hs

```diff
@@ -125,3 +127,21 @@ data TelemetryTimeSpent = TelemetryTimeSpent
 
 instance ToJSON TelemetryTimeSpent where
   toEncoding = genericToEncoding defaultOptions
+
+data CIEnvironment
+  = GithubAction
+  | AzurePipeline
+  | Bamboo
+  | UnknownCI
+  deriving (Eq, Ord, Generic)
+
+-- Modifying values below, also requires modifying value in
+-- FOSSA's server's cli telemetry type for compatibility.
```

Comment:
Where are these mentioned in the server? I don't see them mentioned in https://github.com/fossas/FOSSA/pull/8362/files or elsewhere in the core codebase

Reply:
Ah! good catch - I had a fixed implementation approach, so this comment is not needed anymore. I will remove the comment.